Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,209 @@
+        private const string DownArrow = "\u2193";
+
+        private const char LeftKey = 'A';
+        private const char UpwardKey = 'W';
+        private const char RightKey = 'D';
+        private const char DownKey = 'S';
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Part 1 - Convert with 2 conversion modes: From decimal to binary, From binary to decimal.
+            ConvertNumberSystem();
+
+            //Part 2 - Use your WASD (both capital and non capital) keys to print arrows.
+            ConvertLettersToArrows();
+        }
+
+        private static void ConvertNumberSystem()
+        {
+            var mode = GetConversionMode();
+            var number = GetNumber(mode);
+            var convertedNumber = (mode == 1) ? ConvertDecimalToBinary(int.Parse(number)) : ConvertBinaryToDecimal(number).ToString(); ;
+            Console.WriteLine($"Your {number} was converted to {convertedNumber}.");
+        }
+
+        private static string ConvertDecimalToBinary(int number)
+        {
+            var binary = new StringBuilder();
+            var result = 0;
+
+            do
+            {
+                result = number / 2;
+                binary.Append(number % 2);
+                number = result;
+            } while (result != 0);
+
+            return TruncateLeadingZeros(ReverseNumber(binary.ToString()));
+        }
+
+        private static string ReverseNumber(string number)
+        {
+            var reverseNumber = new StringBuilder();
+
+            for (var i = number.Length - 1; i >= 0; i--)
+            {
+                reverseNumber.Append(number[i]);
+            }
+
+            return reverseNumber.ToString();
+        }
+
+        private static string TruncateLeadingZeros(string number)
+        {
+            var firstOneDigitIndex = number.IndexOf('1');
+
+            return number.Substring(firstOneDigitIndex, number.Length - firstOneDigitIndex);
+        }
+
+        private static int ConvertBinaryToDecimal(string binary)
+        {
+            var number = 0;
+
+            for (var i = 0; i < binary.Length; i++)
+            {
+                number += int.Parse(binary[i].ToString()) * (int)Math.Pow(2, binary.Length - i - 1);
+            }
+
+            return number;
+        }
+
+        private static int GetConversionMode()
+        {
+            Console.WriteLine("The program has 2 conversion modes:");
+            Console.WriteLine("1. From decimal to binary.");
+            Console.WriteLine("2. From binary to decimal.");
+
+            var isValidMode = false;
+            var mode = 0;
+            do
+            {
+                Console.WriteLine("Choose 1 or 2:");
+                isValidMode = int.TryParse(Console.ReadLine(), out mode);
+
+                if (!isValidMode && (mode != 1 && mode != 2))
+                {
+                    isValidMode = false;
+                }
```

Comment:
Here we can clearly see what happens if you choose to ever use other methods/values. That will need a lot of changes. That is why I did point out using constants (for now) could be helpful.